pkg/controller/directvolumemigration/migrate.go

```diff
@@ -22,6 +54,8 @@ func (r *ReconcileDirectVolumeMigration) migrate(direct *migapi.DirectVolumeMigr
 		Owner:            direct,
 		Phase:            direct.Status.Phase,
 		PhaseDescription: direct.Status.PhaseDescription,
+		PlanResources:    planResources,
+		MigrationUID:     string(migration.UID),
```

Comment:
This seems like it would fail if there was no connected migration. @shubham-pampattiwar have you been able to test with an isolated DVM?

Reply:
Specifically, if migration is nil, wouldn't trying to run `string(migration.UID)` cause a panic?

Reply:
`migration` won't ever be `nil` right? L15 sets it to an empty object. My initial readthrough assumed this would set UID to an empty string. @shubham-pampattiwar can you confirm?

Reply:
Can test this on my end as well.

Reply:
Thank you @djwhatle for pointing it out, I was not getting a panic in instantiating task object due to the precise reasons stated by @dymurray But some changes were needed in labeling the destination PVCs. I have updated the PR with the latest changes, please take another look. With these latest changes, I have tested a DVM Migration, DVM Rollback as well as an independent DVM Migrate Operation.